buildenv/jenkins/JenkinsfileBase

```diff
@@ -193,6 +193,37 @@ def setup() {
 			OPENJ9_BRANCH_OPTION = (params.OPENJ9_BRANCH) ? "--openj9_branch ${params.OPENJ9_BRANCH}" : ""
 			OPENJ9_SHA_OPTION = (params.OPENJ9_SHA) ? "--openj9_sha ${params.OPENJ9_SHA}" : ""
 
+			// system test repository exports to be used by systemtest/common.xml
+			if (params.ADOPTOPENJDK_SYSTEMTEST_REPO) {
+				env.OPENJDK_SYSTEMTEST_REPO = "${params.ADOPTOPENJDK_SYSTEMTEST_REPO}" 
```

Comment:
- Could we have the environment variable `OPENJDK_SYSTEMTEST_REPO ` match with the parameter `ADOPTOPENJDK_SYSTEMTEST_REPO`? Having a mismatching name could confuse the users.
- We should be able to have `params.ADOPTOPENJDK_SYSTEMTEST_REPO` directly. There is no need to use `"${ ...}"`

Reply:
Changes made! Thanks for the suggestions.